randovania/games/dread/gui/ui_files/games_tab_dread_widget.ui

```diff
@@ -73,12 +73,12 @@
     <item>
      <spacer name="intro_spacer">
       <property name="orientation">
-       <enum>Qt::Orientation::Vertical</enum>
+       <enum>Qt::Vertical</enum>
```

Comment:
This unqualified enum feels weird. Are you using an older qt creator or something?

Reply:
Thanks for catching that. Turns out i had it both installed for qt6 and qt5, but accidentally kept always using the qt5 one.